.azure-devops/nova-facade-release.yml

```diff
@@ -14,52 +21,71 @@ variables:
   - name: tags
     value: production,externalfacing
   - name: serviceTreeID
-    value: 6F8CD842-E117-412F-BAE4-56A3B6166594
+    value: ade7d667-42f5-485a-91a9-f1dc6482a9b0
   - name: adoNpmFeedBaseUrl
     value: https://pkgs.dev.azure.com/domoreexp/_apis/packaging/feeds/npm-mirror
 
-jobs:
-  - job: compliance
-    displayName: Compliance checks
-    pool:
-      name: 1ES-Teams-Windows-2022-DomoreexpGithub
-    steps:
-      - template: ./steps/service-tree.yml
-        parameters:
-          serviceTreeID: $(serviceTreeID)
-      - template: ./steps/compliance-steps.yml
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
+
+  parameters:
+    sdl:
+      sourceAnalysisPool:
+        name: Azure-Pipelines-1ESPT-ExDShared
+        image: windows-2022
+        os: windows
+    stages:
+      - stage: release
+        variables:
+          # OPTIONAL: Set this varibale to 'true' to enable signing in a target stage.
+          # Remove if signing is not required.
+          Build.ESRP.CodeSign.Enabled: true
+          # OPTIONAL: To disable required tools not applicable in the pipeline set to false.
+          # Supported values: BinSkim, Roslyn, ESLint, PREFast.
+          Build.SDL.<Roslyn>.Enabled: false
+          Build.SDL.<ESLint>.Enabled: true
+        jobs:
+          - job: compliance
+            displayName: Compliance checks
+            pool:
+              name: Azure-Pipelines-1ESPT-ExDShared
+              image: windows-2022
+              os: windows
+            steps:
+              - template: .azure-devops/steps/service-tree.yml@self
```

Comment:
question: isnt this something which is done out of the box in official template? In vscode I am getting warning: 
![image](https://github.com/user-attachments/assets/a317e686-9241-4ade-9619-4eba3bcad653)

Reply:
After remove it. I has slighly different folder structure in SBom-X. 
![image](https://github.com/user-attachments/assets/35e49187-1ae1-4aaf-b140-b25bb2d436c1)

Reply:
After discussing with Mikhail we've figured out the manifest it's not necessary for publishing to npm, so as long as the pipeline it's compliant, we should be good.